src/commands/functions/functions-create.js

```diff
@@ -128,8 +132,9 @@ const formatRegistryArrayForInquirer = function (lang) {
 /**
  * pick template from our existing templates
  * @param {import('commander').OptionValues} config
+ *
  */
-const pickTemplate = async function ({ language: languageFromFlag }) {
+const pickTemplate = async function ({ language: languageFromFlag }, isEdgeFunc) {
```

Comment:
Rather than a `isEdgeFunc` boolean, do you think we could we have a `type` enum variable containing `serverless` or `edge`? I feel that could be extended to contain other types in the future.